Appsflyer doc hotfix (#1073)
rudderlabs/rudderstack-docs: docs/destinations/streaming-destinations/appsflyer.mdx

<div class="infoBlock">
It is highly recommended to use the cloud mode to send events to AppsFlyer in the following scenarios:
<ul>
<li>If you want to use RudderStack's <Link to="/features/transformations/">Transformations</Link> feature.</li> 
<li>If you want to send server-side events.</li>
</ul>
</div>

Once you have confirmed that the source platform supports sending events to AppsFlyer, follow these steps:

1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **AppsFlyer**.
2. Assign a name to the destination and click **Continue**.

### Connection settings

The following connection settings are required to configure AppsFlyer as a destination in RudderStack:

- **AppsFlyer Dev Key**: Refer to the <a href="#faq">FAQ</a> section below for more information on obtaining your AppsFlyer dev key.
- **App ID**: Enter your Apple or Android app ID.
  - **Android App ID**: This the application ID used in your `app/build.gradle` file.
  - **Apple App ID**: This is the iTunes Application ID and it is mandatory for the iOS applications.

<div class="infoBlock">

> If you are connecting a source other than Android or Apple (iOS, watchOS, iPadOS, tvOS), you need to add a user transformation that adds the OS information required in the events. Refer to the <a href="#connecting-appsflyer-to-non-androidios-sources">Connecting AppsFlyer to non-Android/iOS sources</a> section below for more information.

Author: ```suggestion
To connect a source other than Android or Apple (iOS, watchOS, iPadOS, tvOS), you need to add a user transformation that adds the OS information required in the events. Refer to the <a href="#connecting-appsflyer-to-non-androidios-sources">Connecting AppsFlyer to non-Android/iOS sources</a> section below for more information.
```

</div>

- **Use Rich Event Names**: Enable this setting to include your app's screen or page name in the <Link to="/event-spec/standard-events/screen">`screen`</Link> or <Link to="/event-spec/standard-events/page">`page`</Link> event names.
- **Sharing Filter**: By default, the value for this setting is set to `all`. You can use this setting to meet any regulatory requirements like GDPR and CCPA, complying with user opt-out mechanisms, and for any other business use-case. For more information on this setting, refer to the <a href="https://support.appsflyer.com/hc/en-us/articles/207034486-Server-to-server-events-API-for-mobile-S2S-mobile-#sharing_filter-16">AppsFlyer Help Center</a>.

> - **Client-side Event Filtering**: This setting is applicable only if you are sending events to AppsFlyer via device mode. It lets you specify which events should be blocked or allowed to flow through to AppsFlyer. Refer to the <Link to="/sources/event-streams/sdks/event-filtering/">Client-side Event Filtering</Link> guide for more information on this setting.

Author: ```suggestion
- **Client-side Event Filtering**: This setting is applicable **only if** you are sending events to AppsFlyer via device mode. It lets you specify which events should be blocked or allowed to flow through to AppsFlyer. Refer to the <Link to="/sources/event-streams/sdks/event-filtering/">Client-side Event Filtering</Link> guide for more information on this setting.
```

<div class="infoBlock">

For more information on this field, refer to the <a href="https://support.appsflyer.com/hc/en-us/articles/207034486-Server-to-server-events-API-for-mobile-S2S-mobile-#sharing_filter-16">AppsFlyer Help Center</a> page.
</div>

- **Client-side Event Filtering**: This setting is applicable only if you are sending events to AppsFlyer via device mode. 
- **Client-side Event Filtering**: This setting is applicable **only if** you are sending events to AppsFlyer via device mode. It lets you specify which events should be blocked or allowed to flow through to AppsFlyer. Refer to the <Link to="/sources/event-streams/sdks/event-filtering/">Client-side Event Filtering</Link> guide for more information on this setting.

<div class="infoBlock">
  
Refer to the <Link to="/sources/event-streams/sdks/event-filtering/">Client-side Event Filtering</Link> guide for more information on this setting.
</div>
- **Status Callback URLs**: Specify the callback URLs to be used for user deletion requests. You can provide multiple callback URLs by separating them by a comma.

> - **Use native SDK to send events**: By default, this setting is enabled. To send events via cloud mode or device mode, you can toggle this setting on or off.

Reviewer: added

### Deleting a user

<div class="warningBlock">
To delete a user in AppsFlyer, you must configure the <strong>Status Callback URLs</strong> and <strong>API Token</strong> settings in the RudderStack dashboard.
</div>

> 

Reviewer 2: Can we also talk about api-token

Reviewer: @utsabc API token is already mentioned as being mandatory in the warning block. Can you please clarify what else should be modified?

Reviewer 2: Oh oops I missed it nps
